area/midgaard.are

```diff
@@ -2744,7 +2776,50 @@ officer blade~
 You see a sword of superior craftsmanship. It looks like something a soldier 
 might use.
 ~
-
+#3388
+shield sun~
+the Arbiter's shield~
+A shield embossed with the image of the sun lies here.~
+bronze~
+armor ABG AJ
+24 24 24 20 0
+86 18 3000 P
+E
+shield~
+This bronze shield has been embossed with the symbol of  sun. Its outer edge
+has been polished and sharpened such that it probably requires greater skill
+to master in combat compared to more common designs.
+~
+A
+18 2
+A
+19 2
+A
+23 5
+#3389
+toga linen~
+a fine linen toga~
+A fine linen toga lies in a crumpled heap.
+~
+cloth~
+armor AG AD
+20 20 20 28 0
```

Comment:
+20 armour is lower than usual for a level 85 item, but the +28 magic is quite high, and "rainbow resists" for the other magic types makes this potentially quite a good item for some encounters.

Reply:
:+1: